
 This patch removes an unfortunate piece of debug code that crept into
 the 2.4.1 release and causes it to purposely, but incorrectly seg fault
 whenever a volume name error occurs.  Typically this may happen if a
 non-bacula tape is mounted at startup or at the end of volume during
 restore.

 Apply the patch with:

 cd <bacula-source>
 patch -p0 <2.4.1-sd-crash.patch
 ./configure <your-options>
 make
 ...
 make install



Index: src/stored/acquire.c
===================================================================
--- src/stored/acquire.c	(revision 7405)
+++ src/stored/acquire.c	(working copy)
@@ -266,7 +266,6 @@
             dev->close();                                                          
          }
          dev->set_load();
-         ASSERT(0);
          /* Fall through */
       default:
          Jmsg1(jcr, M_WARNING, 0, "%s", jcr->errmsg);
